tools/c7n_org/c7n_org/utils.py

```diff
@@ -8,9 +8,11 @@ def account_tags(account):
     for t in account.get('tags', ()):
         if ':' not in t:
             continue
-        k, v = t.split(':', 1)
-        k = 'Account%s' % k.capitalize()
-        tags[k] = v
+        parts = t.split(':')
```

Comment:
this is all about supporting tags with multiple `:` , could you add a description to the pull request readme? this feels like a change that could use a unit test as the behavior change is subtle, undocumented, and would potentially break in a refactoring without an accompanying test.